kolibri/plugins/coach/assets/src/vue/coach-exercise-render-page/page-status/index.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+
+  <div id="page-status">
+    <div class="column pure-u-3-4">
+      <div id="user-name-container">
+        <mat-svg
+          id="svg-item"
+          category="action"
+          name="face"
+        />
+        <h1 id="user-name">{{ userName }}</h1>
+      </div>
+      <div id="class-name">
+        <content-icon class="svg-icon" :kind="exerciseKind"/>
+        {{ contentName }}
+      </div>
+      <div id="assessment">
+        <progress-icon class="svg-icon" :progress="progress"/>
+        {{ $tr('mastered') }} : {{ assessment }} {{ $tr('assessment') }}
```

Comment:
this needs to all be one translation string - we can't use string concatenation on user-facing text like this